src/screens/TimeCounterScreen.tsx

```diff
@@ -31,6 +31,9 @@ export const TimeCounterScreen = (props: Props) => {
   );
 
   useEffect(() => {
+    props.navigation.setOptions({
```

Comment:
should be set in screen.stack options https://reactnavigation.org/docs/headers/ 
![Zrzut ekranu 2020-09-7 o 12 16 49](https://user-images.githubusercontent.com/67273035/92377229-1dc78680-f104-11ea-8e04-44e643b48046.png)